CMakeLists.txt

```diff
@@ -188,7 +189,7 @@ if (APPLE)
 
       set(HAS_AUDIOFILE TRUE)            # Requires libsndfile
       set(MAC_APPCAST_URL "")
-
+      set(USE_COREAUDIO 1)
```

Comment:
sure about this? Not depending on `BUILD_COREAUDIO`

Reply:
Removed in 8504c0f. Proper use is further down in the file.